cdap-common/src/main/java/io/cdap/cdap/common/internal/DefaultLocalFileProvider.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.common.internal;
+
+import java.io.File;
+
+/**
+ * Provides a File object using local files.
+ */
+public class DefaultLocalFileProvider implements LocalFileProvider {
+
+  @Override
+  public File createNewFile(String name) {
+    return new File(name);
```

Comment:
This implementation doesn't seem to match with the interface description. Is it supposed to create a new file? Also, what is the base path? Or does it expect the `name` always be the full path?